### ISSUE TITLE
According to https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html, ckeditor styling options should be coded as buttons instead of anchor

### ISSUE THREAD

**Issue by ananthula** [ckeditor/ckeditor4#5480]
## Accessibility

Bug

CKeditor toolgroup has many options to style the content(bold, italics etc). Internally they are coded as anchor elements but accoring to https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html they should be buttons.

Open ckeditor in the playground and inspect the bold, italic element. They are anchors but should be buttons.

**Comment by Comandeer** [comment]
As it's stated on the linked page:

> If custom controls are created, however, or interface elements are programmed (in code or script) to have a different role and/or function than usual, then additional measures need to be taken to ensure that the controls provide important information to assistive technologies and allow themselves to be controlled by assistive technologies.

These additional measures were taken, e.g. all buttons have an appropriate role of a button and adjusted keyboard handling. Due to that, they're still compliant with the WCAG standard.
